app/views/spree/admin/payments/source_views/_paypal_braintree.html.erb

```diff
@@ -1,10 +1,15 @@
 <fieldset data-hook="credit_card">
+  <legend align="center"><%= t('braintree', scope: 'solidus_paypal_braintree') %></legend>
```

Comment:
I didn't want it to say "Credit Card", because it might be Apple Pay or Paypal instead. But you've prompted me to rewrite it with a better localisation call.